src/web/demo.rkt

```diff
@@ -239,7 +239,17 @@
 
             (eprintf "Job ~a complete\n" hash)
             (hash-remove! *jobs* hash)
-            (semaphore-post sema)])])
+            (semaphore-post sema)])]
+            [(list 'sample hash formula sema)
+             (define test (parse-test formula))
+             (eprintf "Sampling job started on ~a..." formula)
+             (define result (run-herbie 'sample test #:seed seed #:profile? #f #:timeline-disabled? #t))
+             (define pctx (job-result-backend result))
+             (define result-hashtable (hasheq 'points (pcontext->json pctx (context-repr (test-context test)))))
+             (hash-set! *completed-jobs* hash result-hashtable)
```

Comment:
Not a big deal but I notice that in `improve` the `*completed-jobs*` stores the full `job-result`, whereas here you're storing it after JSON-ifying it. Probably better to be uniform.

Reply:
I did the same thing with `'alternatives` working on it now.